CHANGELOG.md

```diff
@@ -7,10 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Fixed
-
-- `opentelemetry-instrumentation-boto3sqs` Make propagation compatible with other SQS instrumentations, add 'messaging.url' span attribute, and fix missing package dependencies.
```

Comment:
Why are you deleting this?

Reply:
It was deleted by mistake while merging the file

Reply:
Restored the entry

Reply:
Please restore it properly. You should just add your PR entry. All entries should be grouped under #Unreleased

Reply:
Can you please check now?

Reply:
Please copy and replace this into your branch https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CHANGELOG.md and add a new entry under https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CHANGELOG.md#fixed

Reply:
Done